app/src/main/java/org/fossasia/susi/ai/MainActivity.java

```diff
@@ -119,7 +120,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     protected void onResume() {
         super.onResume();
-        etMessage.requestFocus();
+        //etMessage.requestFocus();
```

Comment:
@gogeta95 I think this patch is perfectly fine and the merge is justified, no chat app really brings up the keyboard automatically and that shouldn't be the way it should work, commenting it out and leaving a comment there in case we change the way the interaction should take place should be the right step. 

This is the correct fix to the breaking user experience caused in the patch to open up automatically.